erply_api.py

```diff
@@ -67,21 +64,26 @@ class Erply(object):
         ,'getEmployees'
         ,'getProducts'
         ,'getProductCategories'
-        ,'getProductCostForSpecificAmount'     # untested
+        ,'getProductCostForSpecificAmount'  
         ,'getProductGroups'
-        ,'getProductPrices'                    # untested, broken ??
-        ,'getProductPriorityGroups'            # untested
-        ,'getProductStock'                     # untested
+        ,'getProductPrices'               
+        ,'getProductPriorityGroups'          
+        ,'getProductStock'                    
         ,'getProductUnits'
         ,'getPurchaseDocuments'
         ,'getReports'
         ,'getSalesDocuments'
         ,'getServices'
         ,'getWarehouses'
+        ,'getAllowedWarehouses'
         ,'verifyUser'
+        ,'getCampaigns'
+        ,'getStocktakings'
+        ,'getAccountStatements'
+        ,'getCostOfGoodsSold'
     )
-    ERPLY_CSV = ('getProductStockCSV', 'getSalesReport')
-    ERPLY_POST = ('saveProduct',)
+    ERPLY_CSV = ('getProductStockCSV', 'getSalesReport','getPurchaseReport')
+    ERPLY_POST = ('saveProduct')
```

Comment:
The removed comma is needed as `ERPLY_POST` variable is filled with tuple of strings. With your change you turned it to just single string..